config.go

```diff
@@ -41,6 +41,9 @@ const (
 
 	// DefaultMaxWritesPerRequest is the default number of writes per request.
 	DefaultMaxWritesPerRequest = 5000
+
+	// DefaultMaxClientConnections is zero, which means no connection limit.
+	DefaultMaxClientConnections = 0
```

Comment:
Do you think there's a reason to make the default something other than 0?
I ask because if something is driving this (like open file handles), should we protect against that in a default installation?

Reply:
The only thing I could think of was that if you were not experiencing a problem, there is really no need to add the complexity.  Other than that, I have no opinion

Reply:
Ok with me. Just wanted to check.